pkg/tsdb/cloudwatch/metric_find_query.go

```diff
@@ -52,7 +52,7 @@ var metricsMap = map[string][]string{
 	"AWS/AutoScaling":       {"GroupDesiredCapacity", "GroupInServiceInstances", "GroupMaxSize", "GroupMinSize", "GroupPendingInstances", "GroupStandbyInstances", "GroupTerminatingInstances", "GroupTotalInstances"},
 	"AWS/Billing":           {"EstimatedCharges"},
 	"AWS/Chatbot":           {"EventsThrottled", "EventsProcessed", "MessageDeliverySuccess", "MessageDeliveryFailure", "UnsupportedEvents"},
-	"AWS/CloudFront":        {"4xxErrorRate", "5xxErrorRate", "BytesDownloaded", "BytesUploaded", "Requests", "TotalErrorRate"},
+	"AWS/CloudFront":        {"4xxErrorRate", "5xxErrorRate", "BytesDownloaded", "BytesUploaded", "Requests", "TotalErrorRate", "Cache hit rate", "Origin latency", "Error rate by status code"},
```

Comment:
These should be the correct [metric names](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/programming-cloudwatch-metrics.html):
```suggestion
	"AWS/CloudFront":        {"4xxErrorRate", "5xxErrorRate", "BytesDownloaded", "BytesUploaded", "Requests", "TotalErrorRate", "CacheHitRate", "OriginLatency"},
```

I can't find the technical metric name corresponding to "Error rate by status code" (see my link to documentation above), so I removed it.

Reply:
ooh... my bad, I copied that from the first section of that topic.
thanks for the review

Reply:
I never use Github's suggestion feature before and just realised that your suggestion can be easily merged without doing something in local.